libs/api/domains/driving-license/src/lib/graphql/models/qualityPhoto.model.ts

```diff
@@ -1,10 +1,16 @@
-import { Field, ObjectType } from '@nestjs/graphql'
+import { Field, ObjectType, ID } from '@nestjs/graphql'
 
 @ObjectType()
 export class QualityPhoto {
```

Comment:
@eirikurn "quality photo" was the translation picked for "gæðamerkt ljósmynd" .. possibly a better name would be "registrationPhoto" or something like that.. apparently there are also photos that are not "gæðamerktar" and the intent is to distinguish between the two in code as well as for the user

Reply:
- I removed the `success` field since it was hardly being used. 
- Removed the id field since its possible to use `CurrentUser` decorator in a resolver field
- renamed to `drivingLicenseQualityPhoto`
- Added error messages if QualityPhoto cant be found and there is a warning filed in the ui
- The query takes a second to fetch the qualityPhoto and @albinagu will come in with a follow up pr with a skeleton loader

Reply:
Nice!